ui/lesson-renderer/src/main/java/org/cru/godtools/tool/lesson/ui/LessonActivity.kt

```diff
@@ -84,6 +84,7 @@ class LessonActivity :
 
     private fun LessonActivityBinding.setupPages() {
         pages.adapter = lessonPageAdapter
+        pages.offscreenPageLimit = 1
```

Comment:
Is the default for this ```0```? And what does this do exactly? I tried searching the codebase on github but only found one other instance of ```pages.offscreenPageLimit```

https://github.com/CruGlobal/godtools-android/blob/d93b9a3ad6aa709906e090c4c6028eff2b58de8c/app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt#L163

Reply:
the default for this is actually `-1` which indicates that `ViewPager2` is going to allow the inner `RecyclerView` to manage when to create/destroy child views. Setting it to a different value causes ViewPager2 to force additional adjust screens to be rendered.

This change allows images to be asynchronously loaded onto a page before it is swiped into view.